components/colorwheel/index.css

```diff
@@ -56,35 +95,69 @@
   }
 }
 
-.spectrum-ColorWheel-gradient {
-  position: relative;
-  z-index: 0;
-  width: var(--spectrum-colorwheel-width);
-  height: var(--spectrum-colorwheel-height);
-  border-radius: 100%;
-  box-sizing: border-box;
-
-  border-width: var(--spectrum-colorwheel-border-size);
-  border-style: solid;
-}
-
-.spectrum-ColorWheel-wheel {
-  position: relative;
-  z-index: 1;
+.spectrum-ColorWheel-internal{
+  display: flex;
+  position: absolute;
+  inset-inline-start: 0;
+  inset-inline-end: 0;
+  inset-block-start: 0;
+  inset-block-end: 0;
+  inline-size: var(--mod-colorwheel-colorarea-container-size, var(--spectrum-colorwheel-colorarea-container-size));
+  block-size: var(--mod-colorwheel-colorarea-container-size, var(--spectrum-colorwheel-colorarea-container-size));
+  margin: auto;
 }
 
-.spectrum-ColorWheel-outerCircle,
-.spectrum-ColorWheel-innerCircle {
-  fill: transparent;
-
-  stroke-width: var(--spectrum-colorwheel-border-size);
+.spectrum-ColorWheel-colorarea-container{
+  block-size: auto;
+  inline-size: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: var(--mod-colorwheel-colorarea-margin, var(--spectrum-colorwheel-colorarea-margin));
 }
 
 .spectrum-ColorWheel-slider {
   @extend %spectrum-ColorControl-hiddenField;
 }
 
 .spectrum-ColorWheel-handle {
-  left: 50%;
+  transform: translate(var(--mod-colorwheel-colorhandle-position, var(--spectrum-colorwheel-colorhandle-position)), 0);
   top: 50%;
+  left: 50%;
+}
+
+/* clip path set border-width wider than than spectrum-colorwheel-wheel to create appreance of a border*/
+.spectrum-ColorWheel-border{
+  position: relative;
+  background-color: var(--mod-colorwheel-border-color, var(--spectrum-colorwheel-border-color));
+  inline-size: var(--mod-colorwheel-width, var(--spectrum-colorwheel-width));
+  block-size: var(--mod-colorwheel-height, var(--spectrum-colorwheel-height));
+  clip-path: path(evenodd, var(--mod-colorwheel-path-borders, var(--spectrum-colorwheel-path-borders)));
+
+  &.is-disabled{
+    background-color: var(--highcontrast-colorwheel-border-color-disabled, var(--mod-colorwheel-fill-color-disabled, var(--spectrum-colorwheel-fill-color-disabled)));
+  }
+}
+
+/* hack: make sure we don't complain about these variables being unused*/
+.🤫 {
+  width: var(--track-width);
+  border-width: var(--border-width);
+}
+
+/* --track-width and --border-width to be used with JS in calculating the clip-path paths and colorarea-container-size */
```

Comment:
do we still have JS calculating anything here? looks like we removed the previous JS from the YML file that was used for come calculations.

Reply:
I did remove a <script> from the .yml, this was doing some calculating for the cavnas element we are no longer displaying and was not related to the paths. 

Since we can't manipulate a string in CSS, I am hard coding the path and providing these values so they can be used in JS without hardcoding.